com.github.rhyme_player.rhyme.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Metainfo isn't specific to flatpak, deb and rpm repositories can benefit from it as well. Consider moving it to your upstream repository.

Reply:
How can I tell electron-builder use them

Reply:
No idea, sorry. Probably it can run some custom commands at deb/rpm package build stage, or has other way to add a custom file to the resulting package?

Reply:
I will check it out thanks anyway